arangod/Agency/PathComponent.cpp

```diff
@@ -22,6 +22,8 @@
 
 #include "PathComponent.h"
 
+#include <iostream>
```

Comment:
Is this intentional? I think it could be removed.

Reply:
Right, that's a backport relic.